Rakefile

```diff
@@ -224,7 +224,15 @@ end
 
 task acceptance: ['smoke:app', 'no_active_record:smoke:app', :cucumber]
 
-task default: [:spec, :acceptance]
+desc "Run short, self-contained snippets"
+task :snippets do
+  sh "echo Running snippets"
+  sh <<-SH
+     ruby snippets/use_active_record_false.rb
```

Comment:
Does this definitely fail if it er... fails?

Reply:
It does, yes.